### DISCUSSION TITLE
problem with libasound2 - I think

### DISCUSSION THREAD

**Discussion by jaykonrad** [kraxarn/spotify-qt#271] [Q&A]
Hi guys - I need some help! I have an old iMac running Ubuntu 18.04. I want to listen to Spotify. So I installed Spotify-QT, connected it to my Spotify (which worked, it is showing me all playlists) and then compiled and installed librespot.

Unfortunately, librespot was in /home/pikewerfer/.cargo/bin - and Spotify-QT would not accept this as a path to librespot. I therefore cp the librespot binary to /home/pikewerfer/target/release/. Now the client starts, and Spotify-QT is happy.

However, when I try to play a song, I get an error message telling me "Failed to resume playback: Player command failed: No active device found". When I check the Spotify-QT settings, it says that the client is running. When I check the Spotify-QT logs, I find three error messages:

This happens first and is in the Spotify log: [13:11:19] [err] /home/pikewerfer/target/release/librespot: error while loading shared libraries: [libasound.so](http://libasound.so/).2: cannot open shared object file: No such file or directory
Then this happens: [13:11:26] [err] Request failed: Error transferring https://api.spotify.com/v1/me/player/play - server replied: Not Found
Then this: [13:11:26] [err] me/player/play failed: Player command failed: No active device found
When I do sudo apt-get install libasound2, Ubuntu tells me I already have the latest package. I am stuck - what else could I do to solve this?

**Comment by kraxarn** [comment]
Sounds more like an issue with librespot rather than spotify-qt, so you will probably have more luck asking there for help. Also consider updating your version of Ubuntu to at least 20.04, as that's the oldest version currently supported by spotify-qt.
